<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>371</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 237, 247);
background-color: qlineargradient(spread:pad, x1:0.506, y1:0.630682, x2:0.511, y2:1, stop:0 rgba(215, 254, 255, 207), stop:1 rgba(123, 209, 205, 232));</string>
  </property>
  <widget class="QPushButton" name="input_browse_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font: bold;
padding-left:20px;
text-align: left;
color: rgb(248, 251, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(97, 227, 221), stop:1 rgb(43, 176, 163));
border: 2px solid #f8fbff;
border-radius: 25px;
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_dir_path">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
text-align: left;
color: rgb(0, 0, 0);
background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_dir_path">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
text-align: left;
color: rgb(0, 0, 0);
background-color: transparent;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="output_browse_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font: bold;
padding-left:20px;
text-align: left;
color: rgb(248, 251, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(97, 227, 221), stop:1 rgb(43, 176, 163));
border: 2px solid #f8fbff;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="OK_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Monospace&quot;;
font: bold;
color: rgb(248, 251, 255);
background-color: rgb(43, 176, 163);
</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Input_dir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font: bold;
text-align: left;
color: rgb(43, 176, 163);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Input Directory</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Scale">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font: bold;
text-align: left;
color: rgb(43, 176, 163);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Scale ratio</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Output_dir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font: bold;
text-align: left;
color: rgb(43, 176, 163);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Output Directory</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="scale_value">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
text-align: left;
color: rgb(0, 0, 0);
background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:right;
padding-right:15px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>&lt;strong&gt;Created by&lt;/strong&gt; Thai Bao Khang</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
